pyproject.toml

```diff
@@ -57,8 +55,10 @@ Homepage = "https://github.com/EleutherAI/lm-evaluation-harness"
 Repository = "https://github.com/EleutherAI/lm-evaluation-harness"
 
 [project.optional-dependencies]
+torch = ["torch>=1.8"]
+hf = ["transformers", "torch>=1.8", "accelerate>=0.26.0", "peft>=0.2.0"]
```

Comment:
two new extras. My guess is that people will generally want these, but pip has no way of specifying a "default" install and a "minimal" install.